plaso/cli/image_export_tool.py

```diff
@@ -313,10 +320,8 @@ def _ExtractWithFilter(
       output_writer.Write(
           'Extracting file entries from: {0:s}\n'.format(display_name))
 
-      environment_variables = self._knowledge_base.GetEnvironmentVariables()
-      filter_file_object = filter_file.FilterFile(filter_file_path)
-      find_specs = filter_file_object.BuildFindSpecs(
-          environment_variables=environment_variables)
+      find_specs = tools.FindSpecsGetter().GetFindSpecs(artifacts_registry,
```

Comment:
It is unclear what `FindSpecsGetter` means, please rename to something that indicates the action of the method, maybe `CreateFilterFindSpecs` or `GetFilterFindSpecs`?

Also please do not chain commands, this makes the code harder to debug.

Reply:
Done.